python/lsst/afw/table/source.cc

```diff
@@ -112,6 +112,8 @@ void declareSourceRecord(PySourceRecord & cls) {
     cls.def("updateCoord",
             (void (SourceRecord::*)(image::Wcs const &, PointKey<double> const &)) &SourceRecord::updateCoord,
             "wcs"_a, "key"_a);
+
+    cls.def_property_readonly("table", &SourceRecord::getTable);
```

Comment:
Please also do this in `peak.cc`

Please consider moving all of these to just under the wrapper for "getTable" so it's clear they are two different ways of getting the same data.

Consider also doing it in `baseColumnView.cc'. Admittedly that is on a column view instead of a record. My argument in favor it makes all objects that have "getTable" the same.